doc/api/globals.md

```diff
@@ -5,6 +5,10 @@
 These objects are available in all modules. Some of these objects aren't
 actually in the global scope but in the module scope - this will be noted.
 
+The objects listed here are specific to Node.js. There is a number of
```

Comment:
I *think* the second sentence should start with "There are a number...", but I'm not sure. Feel free to ignore.